Nanoleaf-Api/NanoleafClient.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+using Nanoleaf_Api.Endpoints;
+using Nanoleaf_Api.Endpoints.Interfaces;
+
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+using RestSharp;
+
+namespace Nanoleaf_Api
+{
+    public interface INanoleafClient
+    {
+        IEffectsEndpoint EffectsEndpoint { get; }
+
+        IAuthorizationEndpoint AuthorizationEndpoint { get; }
+    }
+
+    public class NanoleafClient : INanoleafClient
```

Comment:
Please make a constructor which accepts a token, ip and port, to be used by already saved devices